imageCompress/crop.py

```diff
@@ -1,18 +1,69 @@
-def crop(image, height, width):
+
+import numpy as np
+import os
+from skimage.io import imread, imsave
+
+def crop(img_path, H, W):
     """
     Function to crop images
-
     Parameters:
     ----------
-    image, a 3d array of the image (obtained using plt.imread(image))
-    height, integer, the desired height of the cropped image
-    width, integer, the desired width of the cropped image
-
+    img_path - File path of the image .
```

Comment:
The type of the input parameter (str) should be added.

Reply:
done